EventLite/Data/CatalogSeed.cs

```diff
@@ -53,33 +51,123 @@ private static IEnumerable<CatalogEvent> GetPreconfiguredCatalogEvents()
                     Start = storedStart = DateTime.Today.AddDays(1).AddHours(12),
                     End = storedStart.AddHours(9),
                     PictureUrl = "http://externalcatalogbaseurltobereplaced/api/pic/1",
-
                     VenueName = "Washington State Convention Center",
                     VenueAddressLine1 = "705 Pike St.",
                     VenueCity = "Seattle",
                     VenueStateProvince = "WA",
                     VenuePostalCode = "98101",
                     VenueMapUrl = "http://TODOREPLACEME/",
-
-
-                    // TODO: Understand why this syntax works for Venue property vs.
```

Comment:
Please retain this comment, it is reminder for me to investigate

Reply:
Added comments back in